stl/inc/ostream

```diff
@@ -454,7 +454,14 @@ public:
         return *this;
     }
 
-#if _HAS_CXX17 // LWG-2221 "No formatted output operator for nullptr"
+#if _HAS_CXX23
+    template <class = void> // TRANSITION, ABI
+    basic_ostream& operator<<(const volatile void* _Val) {
+        return *this << (const_cast<const void*>(_Val));

```

Comment:
Style nitpick: Parentheses here are unnecessary. (`const_cast` is a "postfix expression" with extremely high precedence.)